recipes/csxtools/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "csxtools" %}
+{% set version = "0.1.16" %}
+{% set sha256 = "6c7479be91a2980610f065e656a9186ca5c91e863d96b0d7dceb18c497f55873" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NSLS-II-CSX/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - _openmp_mutex  # [linux]
```

Comment:
I think we had to add `_openmp_mutex` for other recipes too, so I assume it's the right thing to do here (self-answering my question from the previous round of review.)

Reply:
This shouldn't be needed 🤔

Reply:
The easiest way to check it is to remove this dependency and try building it again.

@kbeeperez, can you please comment `_openmp_mutex` out or remove it, so that we could see the failure and use the suggestions from CF folks to fix it?